.travis.yml

```diff
@@ -25,3 +25,5 @@ services:
 
 before_script:
   - psql -c 'create database almaren;' -U postgres
+  - psql -U postgres -f src/test/resources/sample_data/column_profile.ddl
```

Comment:
Create a simple table with just 1 or 2 column, don't need to populate...

1. write **sample data** using targetJdbc (maximum 10 records)
2. read the data using sourceJdbc
3. compare the results from souceJdbc with the **sample data**

Reply:
We will use this approach to test any kind of source, so create a function for it.